Dockerfile.rhel

```diff
@@ -0,0 +1,22 @@
+# This Dockerfile is used by CI to publish odo binary artifacts.
+# It builds an image containing the Mac, Win and Linux version of odo binary on the
+# OpenShift golang image.
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.11 AS builder
+LABEL Description="This image is used to build odo(Developer-focused CLI for OpenShift) \
+executables for Windows, macOS and linux" Version="1.0"
+
+COPY . /go/src/github.com/openshift/odo
+WORKDIR /go/src/github.com/openshift/odo
+RUN go get github.com/mitchellh/gox &&\
+    make cross
+
+# For local testing use registry image registry.access.redhat.com/ubi7/ubi instead of
```

Comment:
@mohammedzee1000 Instead of keeping one more docker file i just make a comment of how anybody can use it locally.

Does it make sense ?

Reply:
well ok